e2e/interchaintestv8/operator/operator.go

```diff
@@ -138,7 +138,7 @@ func MembershipProof(trusted_height uint64, paths string, writeFixtureName strin
 
 // UpdateClientAndMembershipProof is a function that generates an update client and membership proof
 func UpdateClientAndMembershipProof(trusted_height, target_height uint64, paths string, args ...string) (*sp1ics07tendermint.IICS02ClientMsgsHeight, []byte, error) {
-	args = append([]string{"fixtures", "update-client-and-membership", "--trusted-block", strconv.FormatUint(trusted_height, 10), "--target-block", strconv.FormatUint(target_height, 10), "--key-paths", paths}, args...)
+	args = append([]string{"fixtures", "update-client-and-membership", "--private-cluster", "--trusted-block", strconv.FormatUint(trusted_height, 10), "--target-block", strconv.FormatUint(target_height, 10), "--key-paths", paths}, args...)
```

Comment:
Hmm, but external people can't run these tests then. Should there be an env variable or something for this in our tests? (To use or not use private cluster)

Reply:
They can't really for most other tests either:
```go
sp1Config := relayer.SP1Config{
    ProverType:     prover,
    PrivateCluster: prover == testvalues.EnvValueSp1Prover_Network,
}
```

Would have to change this there too for that option to make much sense

Reply:
Done